src/templates/Homepage/index.css

```diff
@@ -112,7 +112,7 @@
   }
 }
 
-@media screen and (min-width: 769px) and (max-width: 992px) {
```

Comment:
This 992px breakpoint shouldn't be changed, i don't think - it's based on our breakpoint css tokens
```css
  --ic-breakpoint-xs: 0;
  --ic-breakpoint-sm: 576px;
  --ic-breakpoint-md: 768px;
  --ic-breakpoint-lg: 992px;
  --ic-breakpoint-xl: 1200px;
```

Reply:
if you need it to be bigger, maybe use 1200px?